[BUG] phoenix-evals ci broken on type errors

```python
phoenix_evals-mypy: commands[0] /Users/xandersong/phoenix/packages/phoenix-evals> mypy .
/Users/xandersong/phoenix/.tox/phoenix_evals-mypy/lib/python3.12/site-packages/numpy/ma/extras.pyi:2: error: Cannot find implementation or library stub for module named "numpy.lib.index_tricks"  [import-not-found]
/Users/xandersong/phoenix/.tox/phoenix_evals-mypy/lib/python3.12/site-packages/numpy/ma/extras.pyi:2: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src/phoenix/evals/models/litellm.py:59: error: Module "litellm" does not explicitly export attribute "validate_environment"  [attr-defined]
src/phoenix/evals/models/vertexai.py:9: error: Unused "type: ignore" comment  [unused-ignore]
Found 3 errors in 3 files (checked 21 source files)
phoenix_evals-mypy: exit 1 (22.13 seconds) /Users/xandersong/phoenix/packages/phoenix-evals> mypy . pid=35186
  phoenix_evals-mypy: FAIL code 1 (49.47=setup[3.36]+cmd[23.97,22.13] seconds)
  evaluation failed :( (49.51 seconds)
```

--- a/packages/phoenix-evals/src/phoenix/evals/models/litellm.py
+++ b/packages/phoenix-evals/src/phoenix/evals/models/litellm.py
@@ -56,7 +56,7 @@ class LiteLLMModel(BaseModel):
     def _init_environment(self) -> None:
         try:
             import litellm
-            from litellm import validate_environment
+            from litellm.utils import validate_environment
 
             self._litellm = litellm
             env_info = validate_environment(self.model)

--- a/packages/phoenix-evals/src/phoenix/evals/models/vertexai.py
+++ b/packages/phoenix-evals/src/phoenix/evals/models/vertexai.py
@@ -6,7 +6,7 @@ from typing import TYPE_CHECKING, Any, Dict, Optional
 from phoenix.evals.models.base import BaseModel
 
 if TYPE_CHECKING:
-    from google.auth.credentials import Credentials  # type:ignore
+    from google.auth.credentials import Credentials
 
 logger = logging.getLogger(__name__)
 

